src/app.css

```
@@ -225,3 +225,9 @@
         @apply text-primary text-lg font-semibold tracking-normal;
     }
 }
+
+.highlight {
+    border-radius: 2px;
+    background-color: yellow;
+    color: black;
+}
```

Use Tailwind syntax for this:
```suggestion
@utility highlight {
    @apply text-black bg-amber-300;
}
```
I omitted the border radius in this example and used amber instead of yellow because I thought it was more comfortable to read than just bright yellow.

---

Good catch. I have never changed this since my initial tries. I'm fine with that.